mvc/src/main/java/com/interface21/web/support/PathPatternUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.interface21.web.support;
+
+import java.util.Map;
+
+public class PathPatternUtil {
+
```

Comment:
```suggestion
public final class PathPatternUtil {
    private PathPatternUtil() {}
```

저는 유틸성 클래스의 경우 인스턴스의 생성이 불가능하도록,
생성자를 private 으로 선언해주는 것을 좋아합니다.